src/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.unit.cs

```diff
@@ -454,17 +454,27 @@ public static void FileSystemWatcher_CreateManyConcurrentInstances()
     {
         int maxUserInstances = int.Parse(File.ReadAllText("/proc/sys/fs/inotify/max_user_instances"));
         var watchers = new List<FileSystemWatcher>();
+        var directories = new List<string>();
 
         using (var dir = Utility.CreateTestDirectory())
         {
+            // We shouldn't catch I/O exception here since this is test setup
```

Comment:
I agree with the principal, but I fear this is going to cause a lot more instability.  If even one other piece of code has an open inotify handle (e.g. if xunit is using an FSW somewhere, if another test is running concurrently, etc.), this is going to fail.